code/index.html

```diff
@@ -2,21 +2,130 @@
 <html>
   <head>
     <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=7">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
-    <header>
       <!-- Some kind of logo and navbar -->
-    </header>
+    <header>
+      <nav>
+        <ul>
+          <li><a href="https://www.goodreads.com/work/quotes/3078186-the-hitchhiker-s-guide-to-the-galaxy"> News</a></li>
+          <li><a href="https://www.goodreads.com/work/quotes/3078186-the-hitchhiker-s-guide-to-the-galaxy">Contact</a></li>
+          <li><a href="https://www.goodreads.com/work/quotes/3078186-the-hitchhiker-s-guide-to-the-galaxy">About</a></li>
+      </ul>
+    </nav>
+    <h1 class="site-name">STRAWBERRY NEWS <br> 
+     <span> Tastiest news of the week</span></h1>
```

Comment:
Are there any reason you go for a span-tag here. I think it can work better with a text-tag, like a paragraph-tag?

Reply:
i used a <span> to get rid of the gap that i got when using a h1. But maybe i could have done it another way and used an h1. Do you have any good idea about this?